src/components/v5/common/CompletedAction/partials/StreamingPayment/partials/FinalizeWithPermissionsInfo/FinalizeWithPermissionsInfo.tsx

```diff
@@ -0,0 +1,131 @@
+import { isToday, isYesterday } from 'date-fns';
+import React, { type FC } from 'react';
+import { FormattedDate, defineMessages } from 'react-intl';
+
+import PermissionRow from '~frame/v5/pages/VerifiedPage/partials/PermissionRow/index.ts';
+import { formatText } from '~utils/intl.ts';
+import MenuWithStatusText from '~v5/shared/MenuWithStatusText/index.ts';
+import { StatusTypes } from '~v5/shared/StatusText/consts.ts';
+import UserPopover from '~v5/shared/UserPopover/UserPopover.tsx';
+
+import { type FinalizeWithPermissionsInfoProps } from './types.ts';
+
+const displayName =
+  'v5.common.CompletedAction.partials.FinalizeWithPermissionsInfo';
+
+const MSG = defineMessages({
+  todayAt: {
+    id: `${displayName}.todayAt`,
+    defaultMessage: 'Today at',
+  },
+  yestardayAt: {
+    id: `${displayName}.yestardayAt`,
+    defaultMessage: 'Yesterday at',
+  },
+  at: {
+    id: `${displayName}.at`,
+    defaultMessage: 'at',
+  },
+});
+
+const formatDate = (value: string | undefined) => {
+  if (!value) {
+    return undefined;
+  }
+
+  const date = new Date(value);
+
+  if (isToday(date)) {
+    return (
+      <>
+        {formatText(MSG.todayAt)}{' '}
+        <FormattedDate value={date} hour="numeric" minute="numeric" />
+      </>
+    );
+  }
+
+  if (isYesterday(date)) {
+    return (
+      <>
+        {formatText(MSG.yestardayAt)}{' '}
+        <FormattedDate value={date} hour="numeric" minute="numeric" />
+      </>
+    );
+  }
+
+  return (
+    <>
+      <FormattedDate value={date} day="numeric" month="short" year="numeric" />{' '}
+      {formatText(MSG.at)}{' '}
+      <FormattedDate value={date} hour="numeric" minute="numeric" />
+    </>
+  );
+};
+
+const FinalizeWithPermissionsInfo: FC<FinalizeWithPermissionsInfoProps> = ({
```

Comment:
This shouldn't be called FinalizeWithPermissionsInfo, it shows the stream creation